packages/ivass-certified-attributes-importer/test/processor.test.ts

```diff
@@ -0,0 +1,767 @@
+  genericLogger,
+} from "pagopa-interop-commons";
+import { Tenant, unsafeBrandId } from "pagopa-interop-models";
+import { TenantProcessService } from "../src/service/tenantProcessService.js";
+import { ReadModelQueries } from "../src/service/readModelQueriesService.js";
+import { importAttributes } from "../src/service/processor.js";
+import {
+  ATTRIBUTE_IVASS_INSURANCES_ID,
+  downloadCSVMock,
+  downloadCSVMockGenerator,
+  getAttributeByExternalIdMock,
+  getIVASSTenantsMock,
+  getTenantByIdMock,
+  getTenantByIdMockGenerator,
+  getTenantsMockGenerator,
+  getTenantsWithAttributesMock,
+  internalAssignCertifiedAttributeMock,
+  internalRevokeCertifiedAttributeMock,
+  persistentTenant,
+  persistentTenantAttribute,
+} from "./helpers.js";
+
+describe("IVASS Certified Attributes Importer", () => {
+  const tokenGeneratorMock = {} as InteropTokenGenerator;
+  const refreshableTokenMock = new RefreshableInteropToken(tokenGeneratorMock);
+  const tenantProcessMock = new TenantProcessService("url");
+  const csvDownloaderMock = downloadCSVMock;
+  const readModelClient = {} as ReadModelRepository;
+  const readModelQueriesMock = new ReadModelQueries(readModelClient);
+
+  const run = () =>
+    importAttributes(
+      csvDownloaderMock,
+      readModelQueriesMock,
+      tenantProcessMock,
+      refreshableTokenMock,
+      10,
+      "ivass-tenant-id",
+      genericLogger
+    );
+
+  const interopToken: InteropToken = {
+    header: {
+      alg: "algorithm",
+      use: "use",
+      typ: "type",
+      kid: "key-id",
+    },
+    payload: {
+      jti: "token-id",
+      iss: "issuer",
+      aud: ["audience1"],
+      sub: "subject",
+      iat: 0,
+      nbf: 0,
+      exp: 10,
+      role: "role1",
+    },
+    serialized: "the-token",
+  };
+  const generateInternalTokenMock = (): Promise<InteropToken> =>
+    Promise.resolve(interopToken);
+
+  const refreshableInternalTokenSpy = vi
+    .spyOn(refreshableTokenMock, "get")
+    .mockImplementation(generateInternalTokenMock);
+
+  const internalAssignCertifiedAttributeSpy = vi
+    .spyOn(tenantProcessMock, "internalAssignCertifiedAttribute")
+    .mockImplementation(internalAssignCertifiedAttributeMock);
+  const internalRevokeCertifiedAttributeSpy = vi
+    .spyOn(tenantProcessMock, "internalRevokeCertifiedAttribute")
+    .mockImplementation(internalRevokeCertifiedAttributeMock);
+
+  const getIVASSTenantsSpy = vi
+    .spyOn(readModelQueriesMock, "getIVASSTenants")
+    .mockImplementation(getIVASSTenantsMock);
+  const getTenantsWithAttributesSpy = vi
+    .spyOn(readModelQueriesMock, "getTenantsWithAttributes")
+    .mockImplementation(getTenantsWithAttributesMock);
+  const getTenantByIdSpy = vi
+    .spyOn(readModelQueriesMock, "getTenantById")
+    .mockImplementation(getTenantByIdMock);
+  const getAttributeByExternalIdSpy = vi
+    .spyOn(readModelQueriesMock, "getAttributeByExternalId")
+    .mockImplementation(getAttributeByExternalIdMock);
```

Comment:
not required to close this PR, but now that the job is in the monorepo it would be nice to use integration tests like other services

Reply:
@MalpenZibo we could track this in improvements, a generic task to reuse integration testing for all the migrated jobs/scripts

Reply:
I created this https://pagopa.atlassian.net/browse/IMN-875 to track this test and the one in the ANAC job. We could use that task to track also other test rework